File: java2csharp/java/Microsoft.Mobile.Engagement/engagement/reach/activity/EngagementTextAnnouncementActivity.java
/*
 * Copyright (c) Microsoft Corporation. All rights reserved.
 */

package com.microsoft.azure.engagement.reach.activity;

/**
 * Activity displaying a plain text Engagement announcement. Add this in the AndroidManifest.xml
 * file to use it:
 * 
 * <pre>
 * {@code <activity
 *   android:name="com.microsoft.azure.engagement.reach.activity.EngagementTextAnnouncementActivity"
 *   android:theme="@android:style/Theme.Light">
 *     <intent-filter>
 *       <action android:name="com.microsoft.azure.engagement.reach.intent.action.ANNOUNCEMENT" />
 *       <category android:name="android.intent.category.DEFAULT" />
 *       <data android:mimeType="text/plain" />
 *     </intent-filter>
 * </activity>}
 * </pre>
 */
public class EngagementTextAnnouncementActivity extends EngagementAnnouncementActivity
{
  @Override
  protected String getLayoutName()
  {
    return "engagement_text_announcement";
  }
}

File: java2csharp/csharp/Microsoft.Mobile.Engagement/engagement/reach/activity/EngagementTextAnnouncementActivity.cs
/*
 * Copyright (c) Microsoft Corporation. All rights reserved.
 */

namespace com.microsoft.azure.engagement.reach.activity
{

	/// <summary>
	/// Activity displaying a plain text Engagement announcement. Add this in the AndroidManifest.xml
	/// file to use it:
	/// 
	/// <pre>
	/// {@code <activity
	///   android:name="com.microsoft.azure.engagement.reach.activity.EngagementTextAnnouncementActivity"
	///   android:theme="@android:style/Theme.Light">
	///     <intent-filter>
	///       <action android:name="com.microsoft.azure.engagement.reach.intent.action.ANNOUNCEMENT" />
	///       <category android:name="android.intent.category.DEFAULT" />
	///       <data android:mimeType="text/plain" />
	///     </intent-filter>
	/// </activity>}
	/// </pre>
	/// </summary>
	public class EngagementTextAnnouncementActivity : EngagementAnnouncementActivity
	{
	  protected internal override string LayoutName
	  {
		  get
		  {
			return "engagement_text_announcement";
		  }
	  }
	}

}